llvm/test/DebugInfo/Generic/debug-label-opt.ll

```diff
@@ -3,6 +3,12 @@
 ; REQUIRES: asserts
 ; RUN: llc -debug-only=isel %s -o /dev/null 2> %t.debug
 ; RUN: cat %t.debug | FileCheck %s --check-prefix=CHECKMI
+;; FIXME: RemoveDIs debug output is slightly different, e.g.:
```

Comment:
Is this still broken?

Reply:
Nope, I must've dropped a commit that removed this comment...